app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -1,5 +1,6 @@
 module UspsInPersonProofing
   class Proofer
+    mattr_reader :token, :token_expires_at
```

Comment:
Would it be safe to use `Rails.cache` for this to share across instances?

Reply:
I think so but is there a benefit to that over a class member?

Reply:
Class members can be hard to test/reason about sometimes `mattr_reader` vs `cattr_reader` and then have to be reset explicitly in tests, but `Rails.cache` has easy methods for clearing it.

Also with like 10+ boxes, with probably 4+ processes each, each one will have to cache separately, so having a shared cache (we use Redis to back our `Rails.cache`) might save extra requests to the API overall

Reply:
Oh cool it's shared across processes and instances? Yeah I'll go ahead and switch it to `Rails.cache`

Reply:
[in prod, yes!](https://github.com/18F/identity-idp/blob/b0e1dbae1bec3ddca8114888c419bad48f9fbb16/config/environments/production.rb#L3), locally it might just be in-memory